### PR TITLE
Yii2 smarty compatible with Smarty 2

### PR DESCRIPTION
**Pull request by antonmarin**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | no
| Fixed issues  | #17 

almost compatible. registerClass left. Need help

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@ phpunit.phar
 /tests/docker
 /tests/dockerids
 
+/.idea
```
Comment (cebe) [inline .gitignore]: this should go into your gobal gitignore http://stackoverflow.com/questions/7335420/global-git-ignore#7335487

Comment (antonmarin) [inline .gitignore]: removed

**Comment by samdark** [comment]
Inheritance tests should be excluded when running with 2.x since, as I remember, it was not supported in 2.x.

**Comment by samdark** [comment]
@antonmarin what's the problem with `registerClass`?

**Comment by antonmarin** [comment]
No idea on how to implement. Hasnt found any equal method

**Comment by samdark** [comment]
There's no such ability in Smarty 2. That's new for v3.

**Comment by antonmarin** [comment]
Seems like the only way to support v2 is to implement `registerClass`. Will try once more next weekend

**Comment by antonmarin** [comment]
seems like it is not available to make it compatible

**Comment by samdark** [comment]
Yeah...

**Comment by antonmarin** [comment]
So, remove smarty 2.6.0 from composer, please

**Pull request closed without merging**